src/rpc/conn_pool.rs

```diff
@@ -249,7 +246,7 @@ impl ConnPool {
         Ok(())
     }
 
-    pub fn get(&self) -> Result<Arc<Connection>> {
+    pub async fn get(&self) -> Result<Arc<Connection>> {
```

Comment:
Is this async necessary? No await is found in the code block of `get`.

Reply:
Fixed